src/renderer/components/table/table.tsx

```diff
@@ -171,9 +171,20 @@ export class Table extends React.Component<TableProps> {
     });
   }
 
-  renderRows() {
-    const { sortable, noItems, children, virtual, customRowHeights, rowLineHeight, rowPadding, items, getTableRow, selectedItemId, className } = this.props;
+  getContent() {
```

Comment:
Sure... for react components I haven't been able to figure out why `private` or `protected` are good ideas for the instance methods. Because it is not like we call the constructor ourselves.